docs/source/getting_started.rst

```diff
@@ -29,4 +29,12 @@ Each time you want to use the package, just activate the conda environment:
 
     $ conda activate gammasim-tools-dev
 
+If you need access to the DB, ask one of the developers, update the login \
+credentials in set_env_db_template.sh and source it to activate the \
```

Comment:
I thought you would want that users copy the template file to `set_env_db.sh` and not modify directly the template? Otherwise git will always complaint.

Reply:
thats a better idea. thanks